agent/lib_phpunit.c

```diff
@@ -665,6 +666,22 @@ static int nr_phpunit_are_statuses_valid(TSRMLS_D) {
   return 1;
 }
 
+void nr_phpunit_version() {
+  char* string = "PHPUnit\\Runner\\Version::id();";
+  zval retval;
+  int result
+      = zend_eval_string(string, &retval, "Retrieve PHPUnit Version");
+
+  // Add php package to transaction
+  if (result == SUCCESS) {
+    if (Z_TYPE(retval) == IS_STRING) {
+      char* version = Z_STRVAL(retval);
+      nr_txn_add_php_package(NRPRG(txn), "phpunit/phpunit", version);
+      zval_dtor(&retval);
+    }
```

Comment:
```suggestion
    }
    zval_dtor(&retval);
```
Even if the version is somehow not a string, we still want to free the zval